tests/certbot-boulder-integration.sh

```diff
@@ -15,8 +15,10 @@ command -v python > /dev/null || (echo "Error, python executable is not in the P
 
 . ./tests/integration/_common.sh
 export PATH="$PATH:/usr/sbin"  # /usr/sbin/nginx
+CURRENT_DIR="$(pwd)"
 
 cleanup_and_exit() {
+    cd $CURRENT_DIR
```

Comment:
Sorry to keep adding minor things in separate reviews, but I thought the unreported failures in our tests was really bad so I went digging. It turns out the source of the problem is this line. It should be moved below `EXIT_STATUS=$?` so we're capturing the exit status of the last command ran rather than `cd`.

I tested this at https://travis-ci.com/certbot/certbot/builds/102535040 with the prior config directory using files and not symlinks and the tests properly failed.

Reply:
No problem. In particular if it is my modifications that started this ^^

I changed the order of the statements, to get the correct exit code.